src/storage/hummock_test/src/test_utils.rs

```diff
@@ -144,9 +146,52 @@ pub async fn prepare_local_version_manager(
     );
 
     tokio::spawn(
-        HummockEventHandler::new(local_version_manager.clone(), event_rx)
-            .start_hummock_event_handler_worker(),
+        HummockEventHandler::new(
+            local_version_manager.clone(),
+            event_rx,
+            Arc::new(RwLock::new(HummockReadVersion::new(
+                local_version_manager.get_pinned_version(),
+            ))),
+        )
+        .start_hummock_event_handler_worker(),
     );
 
     local_version_manager
 }
+
+pub async fn prepare_local_version_manager_new(
```

Comment:
What's the difference between this function and the original `prepare_local_version_manager`?

Reply:
Mainly to expose event_tx for HummockStorage::for_test.

and the function `prepare_local_version_manager_new ` not init the event_handler thread

Reply:
I see. We may remove one of them after we implement the read-upload separation is finished.